app/controllers/games_controller.rb

```diff
@@ -31,6 +31,17 @@ def update
     if @game.valid? && current_player != @game.white_player
       @game.update(black_player: current_player, state: 1)
       @game.add_pieces_to_board
+
+      ActionCable.server.broadcast('game',
```

Comment:
Be careful here. This might broadcast to all games. You may need to narrow down the broadcast and channel to include the game_id so that way all people playing the game don't collide.